src/online2/online-nnet3-decoding.h

```diff
@@ -0,0 +1,132 @@
+// online2/online-nnet3-decoding.h
+
+// Copyright 2014  Johns Hopkins University (author: Daniel Povey)
+//           2016  Api.ai (Author: Ilya Platonov)
+
+// See ../../COPYING for clarification regarding multiple authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY IMPLIED
+// WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABLITY OR NON-INFRINGEMENT.
+// See the Apache 2 License for the specific language governing permissions and
+// limitations under the License.
+
+
+#ifndef KALDI_ONLINE2_ONLINE_NNET3_DECODING_H_
+#define KALDI_ONLINE2_ONLINE_NNET3_DECODING_H_
+
+#include <string>
+#include <vector>
+#include <deque>
+
+#include "../nnet3/online-nnet3-decodable-simple.h"
```

Comment:
FYI: 
$ grep "\.\." -R . | grep "#include"
./fstext/fstext-utils.h:#include "../base/kaldi-common.h" // for error reporting macros.
./fstext/fstext-utils.h:#include "../util/text-utils.h" // for SplitStringToVector